Components/grid/grid.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+
+/* colors for unit borders */
+:root {
+    --color1: #001219;
+    --color2: #005f73;
+    --color3: #3d9397;
+    --color4: #94d28d;
+    --color5: #e9d8a6;
+    --color6: #ee9800;
+    --color7: #ca6702;
+    --color8: #bb3e03;
+    --color9: #ae2012;
+    --color10: #9b2226;
+    --color11: rgba(255, 255, 255, 0.8);
+}
+
+/* general web-page styles */
+#collections {
+    margin: 0 auto;
+    padding: 0 var(--space-md);
+}
+
+#collections ul{
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+#collections h2 {
+    font-size: calc(2.5 * var(--paragraph));;
+}
+
+
+/* styling the figute containers */
+#collections figure{
+    max-height: 400px;
+    overflow: hidden;
+    position: relative;
+    border-width: 10px;
+    border-style: solid;
```

Comment:
This can be achieved with the shorthand right? `border: 10px solid black`

Reply:
no that is not the case here as I want to use different border colors for each element and I have created separate usage classes for different border colors please go through the complete file you will find these usage classes. this is the reason for not using what you have suggested, Thanks.